Extensions/better_reblogs.js

```diff
@@ -0,0 +1,264 @@
+
+    },
+
+    run_nested: function() {
+        var match_only_main_dash = "ol" ? !XKit.page.peepr : "";
+        XKit.tools.add_css(match_only_main_dash+'.posts .reblog-list {display: none!important} '+
+                           match_only_main_dash+'.posts .reblog-list-item.contributed-content {display: none!important;} '+
+                           match_only_main_dash+'.posts .post_full.post .post_content_inner .post_media ~ .old_reblogs {margin-top: 13px;}',
+                        'better_reblogs');
+        XKit.extensions.better_reblogs.do_nested();
+        XKit.post_listener.add("better_reblogs", XKit.extensions.better_reblogs.do_nested);
+    },
+
+    do_flat: function() {
+        var posts = XKit.interface.get_posts("better-reblogs-done");
+
+        $(posts).each(function() {
+            var $this = $(this);
+            $this.addClass("xkit-better-reblogs-done");
+
+            // trick tumblr into displaying the little blog info popovers for the reblog avatars
+            $this.find(".reblog-avatar").addClass("post_sub_avatar");
+
+            if (XKit.extensions.better_reblogs.preferences.reorder_reblog_title.value){
+                var title = $this.find(".reblog-title");
+                if (!title.length) {
+                    return;
+                }
+                var parent = title.parent();
+                if (!parent.find(".reblog-content").length) {
+                    return;
+                }
+                title.remove();
+                parent.prepend(title);
+            }
+        });
+    },
+
+    do_nested: function() {
+        var posts = XKit.interface.get_posts("xkit-better-reblogs-done");
+
+        $(posts).each(function() {
+            var $this = $(this);
+            $this.addClass("xkit-better-reblogs-done");
+
+            var reblog_tree = $this.find(".reblog-list");
+            var title = reblog_tree.find('.reblog-title');
+
+            if (!reblog_tree.length){
+                return;
+            }
+
+            var cc = $this.find('.contributed-content');
+            if (cc.length) {
+                cc.after('<div class="old_reblogs post_body"></div>');
+                cc.after(title);
+            } else {
+                reblog_tree.after('<div class="old_reblogs post_body"></div>');
+                reblog_tree.after(title);
+            }
+            title.removeClass("reblog-title");
+            title.addClass("post_title");
+
+            var all_quotes = [];
+            reblog_tree.find(".reblog-list-item").each(function() {
+                var $this = $(this);
+                var content = $this.find('.reblog-content');
+                var author = $this.find('.reblog-tumblelog-name');
+                var reblog_data = {
+                    reblog_content: content.html() || '',
+                    reblog_author: author.contents()[0].data || '',
+                    reblog_url: author.href() || ''
```

Comment:
When I try to switch to or enabled the extension with nested style it tells me
![image](https://cloud.githubusercontent.com/assets/1437010/9958775/78d2d884-5e0d-11e5-8834-61f457870d18.png)
.attr("href") confirmed to work